src/main/java/net/sf/jabref/logic/layout/LayoutFormatterPreferences.java

```diff
@@ -45,15 +45,23 @@ public JournalAbbreviationPreferences getJournalAbbreviationPreferences() {
         return journalAbbreviationPreferences;
     }
 
-    public Map<String, String> getCustomExportNameFormatters() {
-        return customExportNameFormatters;
-    }
-
     public FileLinkPreferences getFileLinkPreferences() {
         return fileLinkPreferences;
     }
 
     public JournalAbbreviationLoader getJournalAbbreviationLoader() {
         return journalAbbreviationLoader;
     }
+
+    public void clearCustomExportNameFormatters() {
+        customExportNameFormatters.clear();
+    }
+
+    public void putCustomExportNameFormatter(String formatterName, String contents) {
+        customExportNameFormatters.put(formatterName, contents);
+    }
+
+    public String getCustomExportNameFormatter(String className) {
```

Comment:
Yes. Will change in LayoutEntry as well (I just used the name from refactoring).